packages/frontend-2/assets/css/tailwind-setup/semantic-colors.css

```diff
@@ -1,43 +1,42 @@
+/* stylelint-disable custom-property-empty-line-before */
+/* stylelint-disable comment-empty-line-before */
 /*
   Currently not possible to define different color values for the same color name just through
   the tailwind config, so using CSS variables instead.
-
+  
+  Palette reference: https://tailwindcss.com/docs/customizing-colors
+  
   Only use colors from the default tailwind color palette so that migrating away from this (if ever needed)
-  is easy
+  is easy.
  */
 
 :root {
-  /* background */
-  --theme-color-background: #fff; /* white */
-  --theme-color-background-2: #f3f4f6; /* gray-100 */
-  --theme-color-background-3: #e5e7eb; /* gray-200 */
+  /* blank background */
+  --bg-base: #f1f5f9;
+  /* background for elements that need to sand out from the blank backgorund, e.g. cards. */
+  --bg-default: #fff;
```

Comment:
Ok so "base" is just the base of the page and not ever supposed to be used for any panels on top of it, right? And then "default" is the default BG color for any panels or blocks on top of the base. And then "accent" is like an emphasis state.

Makes sense so far, but I think "default" might be improved considering that we might want to add different variations of colors to use on top of "base". And then it would be useful to organize all of those panel variations together, and make them stand out as a group and not mix with "base" and "accent".

So maybe "default"->"panel", and then we could have "panel-secondary", "panel-ternary" in the future? That way it's immediately clear that "base" and "accent" are their own colors and all of the "panel-*" ones relate to the blocks put on top of "base".

It's also important to evaluate whether or not we will want "accent" to differ depending on the "panel"/"default" variation chosen.

Reply:
```
bg-background -> bg-panel
bg-backgorund-accent -> bg-panel-2
(+ others in the future)

bg-background-page -> bg-page


foreground -> foreground
foreground-dim -> foreground-2
(+ others in the future)

[danger, warning] -> [danger, warning]
[danger, warning]-darker
[danger, warning]-lighter

// Note: we want primary colors to become lighter when focused/hovered in a dark theme, and darker when focused/hovered in a light theme, hence the differentnaming scheme
primary -> primary
primary-focus -> primary
primary-muted -> primary-muted

```

Reply:
Realised a thing: having a `background` color meant that we'd type `bg-background`, `bg-background-2` as actual css class names (`<div class="bg-background-2">`), which triggered me (too many bg-background-backgrounds). Above i was confusing the tailwind class and the css color name 🙃 

`bg-page`/`bg-panel` as a tailwind class would mean `page`/`panel` as a tailwind.config color, which doesn't really work (or felt weird). 

I propose we try to use `base`, `base-2`, and `base-page` as css vars, and added the tailwind.config color `base` with these variations. This means that you can now do `bg-base`, `bg-base-2` and `bg-base-page` when setting element backgrounds.